File: experiment/comments/no_comment.js
function doRangesOverlap(range1, range2){
  return range1.end >= range2.start && range1.start <= range2.end;
}

var scores = {
  start: 1,
  end: 7
}

var desired = {
  start: 5,
  end: 9
}

var result = doRangesOverlap(scores, desired);

File: experiment/comments/no_comment.py
def do_ranges_overlap(range1, range2):
  return (
    range1.get('end') >= range2.get('start')
    and range1.get('start') <= range2.get('end')
  )


scores = {
  'start': 1,
  'end': 7
}

desired = {
  'start': 5,
  'end': 9
}

result = do_ranges_overlap(scores, desired)